source/JsMaterialX/JsMaterialXCore/JsUtil.cpp

```diff
@@ -16,38 +8,27 @@ namespace mx = MaterialX;
 
 extern "C"
 {
-    EMSCRIPTEN_BINDINGS(my_module)
+    EMSCRIPTEN_BINDINGS(util)
```

Comment:
Missing binding: `isValidNamespace`
What about the `hashCombine` func?

Reply:
hashCombine is only used once internally and also not exposed by python so for now I would not expose it